packages/zoe/src/zoe.js

```diff
@@ -214,8 +259,8 @@ import { makeTables } from './state';
  * reallocations that conserve rights and are 'offer-safe', we
  * don't need to do those checks at this step and can assume
  * that the invariants hold.
- * @param  {object[]} offerHandles - an array of offerHandles
- * @returns {TODO}
+ * @param  {OfferHandle[]} offerHandles - an array of offerHandles
+ * @returns {void}
```

Comment:
Is this the right way to say 'undefined'?

Reply:
I couldn't find anything definitive. From the discussion I've seen, I'd prefer to omit it, but VSDoc interprets that as returning `any`, which is way wrong.

I'll use return `undefined`, as more consistent with JavaScript. With that VSCode shows `=> undefined` which isn't far from right.